examples/plugin/src/test/groovy/acceptance/dsl/PetApiDsl.groovy

```diff
@@ -22,7 +22,7 @@ trait PetApiDsl {
             .basePath(context)
             .wireMock(wireMockServer())
             .build()
-        petApi = new PetApi(new ApiClient().setBasePath("${wireMockUrl()}/pet"))
+        petApi = new PetApi(new ApiClient().setServers([new ServerConfiguration("${wireMockUrl()}/pet", "local wiremock instance", [:])]))
```

Comment:
So the behavior has changed a bit with the generator. By default, the generated `ApiClient` creates a `ServerConfiguration` with index `0` with the host `petstore.swagger.io`, defined in `petstore.json` and the default `serverIndex` is `0`. So `basePath` is essentially ignored. So we have to override that `ServerConfiguration` or set `serverIndex` to `null`. I opted for the former